app/api/government_service_data_api/calls_received_metric.rb

```diff
@@ -1,42 +1,63 @@
 class GovernmentServiceDataAPI::CallsReceivedMetric
+  include GovernmentServiceDataAPI::MetricStatus
+
   def self.build(data)
+    data ||= {}
     new(
-      total: data['total'],
-      sampled: data['sampled'],
-      sampled_total: data['sampled_total'],
-      get_information: data['get_information'],
-      chase_progress: data['chase_progress'],
-      challenge_a_decision: data['challenge_a_decision'],
-      other: data['other'],
+      total: data.fetch('total', NOT_APPLICABLE),
+      sampled: data.fetch('sampled', NOT_APPLICABLE),
+      sampled_total: data.fetch('sampled_total', NOT_APPLICABLE),
+      get_information: data.fetch('get_information', NOT_APPLICABLE),
+      chase_progress: data.fetch('chase_progress', NOT_APPLICABLE),
+      challenge_a_decision: data.fetch('challenge_a_decision', NOT_APPLICABLE),
+      other: data.fetch('other', NOT_APPLICABLE),
     )
   end
 
   def initialize(total: nil, sampled: nil, sampled_total: nil, get_information: nil, chase_progress: nil, challenge_a_decision: nil, other: nil)
-    @total = total || 0
-    @sampled = sampled || false
-    @sampled_total = sampled_total || 0
-    @get_information = get_information || 0
-    @chase_progress = chase_progress || 0
-    @challenge_a_decision = challenge_a_decision || 0
-    @other = other || 0
+    @total = total || NOT_PROVIDED
+    @sampled = sampled || NOT_PROVIDED
```

Comment:
I think this might be the issue that I commented earlier, with `sampled` data being displayed. If sampled is `false`, then it's always be set to `NOT_PROVIDED` which will evaluate as truthy.